sc/social/like/subscribers.py

```diff
@@ -151,3 +152,35 @@ def check_sharing_best_practices(obj, event):
         validate_og_lead_image(image)
     except ValueError as e:
         api.portal.show_message(message=e.message, request=request, type='warning')
+
+
+def prefetch_facebook(obj, event):
+    """Prefetching in object if enable."""
```

Comment:
"Prefetching an object if facebook prefetch option in the control panel configlet is enabled."

Reply:
thank you, that docstring was on my radar but I forgot it at the end.